src/plugins/recordTypes/uoc/fields.js

```diff
@@ -0,0 +1,290 @@
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.title.name',
+                defaultMessage: 'Title',
+              },
+            }),
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        authorizedBy: {
+          [config]: {
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.authorizedBy.name',
+                defaultMessage: 'Authorized by',
+              },
+            }),
+            view: {
+              type: AutocompleteInput,
+              props: {
+                source: 'person/local,person/shared',
+              },
+            },
+          },
+        },
+        authorizationDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.authorizationDate.name',
+                defaultMessage: 'Authorization date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        authorizationNote: {
+          [config]: {
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.authorizationNote.name',
+                defaultMessage: 'Authorization note',
+              },
+            }),
+            view: {
+              type: TextInput,
+            },
+          },
+        },
+        startSingleDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.startSingleDate.name',
+                defaultMessage: 'Start/single date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        endDate: {
+          [config]: {
+            dataType: DATA_TYPE_DATE,
+            messages: defineMessages({
+              name: {
+                id: 'field.uoc_common.endDate.name',
+                defaultMessage: 'End date',
+              },
+            }),
+            view: {
+              type: DateInput,
+            },
+          },
+        },
+        userGroupList: {
+          [config]: {
+            view: {
+              type: CompoundInput,
+            },
+          },
+          userGroup: {
+            [config]: {
+              messages: defineMessages({
+                name: {
+                  id: 'field.uoc_common.userGroup.name',
+                  defaultMessage: 'User',
+                },
+              }),
+              repeating: true,
+              view: {
+                type: CompoundInput,
+                props: {
+                  tabular: true,
+                },
+              },
+            },
+            user: {
+              [config]: {
+                messages: defineMessages({
```

Comment:
Did you take out the fullName message here? I think it should stay, and be "User name". Otherwise, when the field is referenced in other places, it will be unclear -- it will sound like "name of the UOC record", which isn't a thing.